.github/workflows/pr-dev-to-main.yml

```diff
@@ -0,0 +1,33 @@
+name: Create Pull Request from dev to main
+
+on:
+  workflow_dispatch:
+
```

Comment:
You can run this for every branch ever?

Reply:
That doesn't matter, it uses the correct branches on its own, it just needs to run.